tests/services/test_browser.py

```diff
@@ -993,7 +994,8 @@ async def test_generate_service_query_suppress_duplicate_questions():
 async def test_query_scheduler():
     delay = const._BROWSER_TIME
     types_ = {"_hap._tcp.local.", "_http._tcp.local."}
-    query_scheduler = _services_browser.QueryScheduler(types_, delay, (0, 0))
+    cache = DNSCache()
+    query_scheduler = _services_browser.QueryScheduler(cache, types_, delay, (0, 0))
```

Comment:
We need another query scheduler test that interacts with a populated DNSCache before we can merge this